web/src/utils/utils.ts

```diff
@@ -33,4 +33,9 @@ export const findDifferenceBetweenTwoDifferentObject = (object1: any, object2: a
   return differenceKey;
 };
 
+export const getLicencePlatePostFix = (input: string | undefined): any => {
+  const regexMatch = input ? input.match(/(?<=-).*/) : null;
+  return regexMatch ? regexMatch[0] : '';
+};
+
```

Comment:
the reason that this is in two places does hurt my soul quite a bit.  Also what this is doing and what this says its doing is two different things. That also hurts my soul

Reply:
Because one on front-end one on backend😒😒